inc/admin/metaboxes/namespace.php

```diff
@@ -1273,12 +1273,20 @@ function contributor_edit_form( $term ) {
 						<td>
 							<?php if ( $value ) : ?>
 								<img src="<?php echo $value; ?>" id="<?php echo $meta_tags['tag']; ?>-thumbnail" width="120" /> <br />
+							<?php else : ?>
+								<img style="display: none" id="<?php echo $meta_tags['tag']; ?>-thumbnail" width="120" />
 							<?php endif; ?>
+							<br />
```

Comment:
This will fix: https://github.com/pressbooks/pressbooks/issues/2355

Reply:
Tested and it's working as expected now